src/pilot/install-director.sh

```diff
@@ -273,20 +273,6 @@ echo "## Updating .bash_profile..."
 echo "source ~/stackrc" >> ~/.bash_profile
 echo "## Done."
 
-# This hacks in a patch to allow realtime RAID creation.
```

Comment:
In the future, to avoid the issue with patches it may be best to have a test to determine if upstream code contains the patch to ship attempting to apply the patch.  I did this in the pass.  The problem doing this is you forget to remove the patch code :-(

Reply:
+1

Reply:
yeah, was thinking about that.  unfortunately a single .patch file may contain multiple fixes, so you may have some parts of the patch that merge ok, and some that don't, so you can't just skip a whole file if it doesn't apply cleanly.  will give it some thought

Reply:
On top of multiple fixes in a patch .. there is also the case where the implementation upstream of a patch would differ, even slightly, from ours , that s usually the case and the reason we dont assume it made it upstream/ can t exactly diff that.